src/services/DocumentSelectionRequestProcessor.ts

```diff
@@ -319,9 +319,12 @@ export class DocumentSelectionRequestProcessor {
 		selectedDocument: string,
 		countryCode: string,
 	): Promise<string> {
+		const yotiSessionBackoffPeriod = this.environmentVariables.fetchYotiSessionBackoffPeriod()
```

Comment:
If these are environment variables could they not be in the yoti service so we don't need to pass them to ever method?
Does the template need updated now these will be used in more lambdas